bin/web-ext.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+import webExt from '../lib/main.js';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
```

Comment:
Any reason for naming these variables like the (old) built-in node variables?

Reply:
good catch, I did that at the start as a quick fix to get enough of web-ext to run `web-ext run` in the initial set of exploratory changes, but I was planning to remove them along with that "node: false" set in .eslintrc.